main/lsp/notifications/sorbet_workspace_edit.cc

```diff
@@ -67,6 +67,10 @@ void SorbetWorkspaceEditTask::run(LSPTypecheckerDelegate &typechecker) {
     if (latencyTimer != nullptr) {
         // TODO: Move into pushDiagnostics once we have fast feedback.
         latencyTimer->clone("last_diagnostic_latency");
+        // Experiment: Weight by number of edits.
+        for (u4 i = 0; i < updates->editCount; i++) {
+            latencyTimer->clone("last_diagnostic_latency_weighted");
```

Comment:
if it works well this will replace `last_diagnostic_latency`. I'm keeping both so I can compare their output while experimenting.